topics/green-software/index.md

```diff
@@ -0,0 +1,9 @@
+---
+aliases: sustainable-software, green-computing
+display_name: Green Software
+short_description: Software projects that aim to greenify computing
+topic: green-software
+wikipedia_url: https://en.wikipedia.org/wiki/Green_computing
+---
+
+This topic gathers projects that exemplify or help provide green computing. Green software is engineered to reduce energy consumption, which considers factors like algorithmic and language efficiency, networking, storage footprint, compute requirements, and so forth. Some projects follow great green software practices that should be highlighted; others help the rest of the world greenify their own code. The projects collected here are a mix of both.
```

Comment:
I'm unsure but it would sound better

```suggestion
This topic gathers projects that exemplify or help to provide green computing. Green software is engineered to reduce energy consumption, which considers factors like algorithmic and language efficiency, networking, storage footprint, compute requirements, and so forth. Some projects follow great green software practices that should be highlighted; others help the rest of the world greenify their own code. The projects collected here are a mix of both.
```

Reply:
I'll take it, thanks!